src/overlays/gamestates/ovl_select/z_select.c

```diff
@@ -20,10 +20,9 @@ void Select_LoadGame(SelectContext* this, s32 entranceIndex) {
     osSyncPrintf(VT_RST);
     if (gSaveContext.fileNum == 0xFF) {
         Sram_InitDebugSave();
-        gSaveContext.unk_13F6 = gSaveContext.magic;
-        gSaveContext.magic = 0;
-        gSaveContext.unk_13F4 = 0;
-        gSaveContext.magicLevel = gSaveContext.magic;
+        gSaveContext.magicFillTarget = gSaveContext.magic;
+        gSaveContext.magicCapacity = 0;
+        gSaveContext.magicLevel = gSaveContext.magic = 0;
```

Comment:
Should there be a comment like "see similar code in `FileChoose_LoadGame` for what this achieves" ?